CHANGELOG.md

```diff
@@ -4,6 +4,39 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v5.0.0](https://github.com/voxpupuli/puppet-openvmtools/tree/v5.0.0) (2024-03-28)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-openvmtools/compare/v4.0.0...v5.0.0)
+
+**Breaking changes:**
+
+- Remove Fedora 19-25 support [\#85](https://github.com/voxpupuli/puppet-openvmtools/pull/85) ([Valantin](https://github.com/Valantin))
+- Remove Ubuntu 14.04, 16.04 and 18.04 from supported OS [\#76](https://github.com/voxpupuli/puppet-openvmtools/pull/76) ([Valantin](https://github.com/Valantin))
+- Remove Debian 7,8,9 and 10 from supported OS [\#75](https://github.com/voxpupuli/puppet-openvmtools/pull/75) ([Valantin](https://github.com/Valantin))
+
+**Implemented enhancements:**
+
+- Add Fedora 38 & 39 support [\#81](https://github.com/voxpupuli/puppet-openvmtools/pull/81) ([Valantin](https://github.com/Valantin))
+- Add AlmaLinux 8 and Centos/Rocky/OracleLinux/AlmaLinux 9 to supported OS [\#77](https://github.com/voxpupuli/puppet-openvmtools/pull/77) ([Valantin](https://github.com/Valantin))
+- Add Debian 11 and 12 to supported OS [\#74](https://github.com/voxpupuli/puppet-openvmtools/pull/74) ([Valantin](https://github.com/Valantin))
+- Add Ubuntu 22.04 to supported OS [\#73](https://github.com/voxpupuli/puppet-openvmtools/pull/73) ([Valantin](https://github.com/Valantin))
+
+**Fixed bugs:**
+
+- Drop puppetlabs/yumrepo\_core from dependencies [\#84](https://github.com/voxpupuli/puppet-openvmtools/pull/84) ([zilchms](https://github.com/zilchms))
+
+**Closed issues:**
+
+- Unit test refactoring [\#79](https://github.com/voxpupuli/puppet-openvmtools/issues/79)
```

Comment:
I addes skip-changelog here because the related PR is already in the changelog. please regenerate the changelog